packages/sui-ssr/server/utils/factory.js

```diff
@@ -0,0 +1,124 @@
+const INDEX_FILE = 'index.html'
+const INDEX_WITHOUT_THIRD_PARTIES_FILE = 'index_without_third_parties.html'
+const DEFAULT_SITE_HEADER = 'X-Serve-Site'
+const DEFAULT_PUBLIC_FOLDER = 'public'
+const DEFAULT_MULTI_SITE_KEY = 'default'
+const EXPRESS_STATIC_CONFIG = {index: false}
+
+export default ({path, fs, config: ssrConf = {}}) => {
```

Comment:
Moved all the same into a factory that gets all the imported dependencies like the package.json config, so that, code is more reliable and we can test it better.